.woodpecker.star

```diff
@@ -1886,6 +1886,7 @@ def opencloudServer(storage = "decomposed", accounts_hash_difficulty = 4, depend
         "OC_JWT_SECRET": "some-opencloud-jwt-secret",
         "EVENTHISTORY_STORE": "memory",
         "OC_TRANSLATION_PATH": "%s/tests/config/translations" % dirs["base"],
+        "OC_WRITE_BUFFER_DURATION": "0", # Disable write buffer so that test expectations are met in time
```

Comment:
```suggestion
        "OC_WRITE_BUFFER_DURATION": "0",  # Disable write buffer so that test expectations are met in time
```

For what ever reason the `check-starlark` job whats you to insert that space there.